Source/WebCore/Modules/webaudio/BiquadDSPKernel.cpp

```diff
@@ -36,6 +36,12 @@
 #include <limits.h>
 #include <wtf/Vector.h>
 
+#if CPU(X86_SSE2)
+#include <immintrin.h>
+#elif HAVE(ARM_NEON_INTRINSICS)
```

Comment:
I think this would be better as a separate `#if`. No real reason to take advantage of the fact that these happen to be mutually exclusive in the includes part even though we will do that below in the code.

Just something that’s on my mind when I think about maintaining includes in various source files.

Reply:
Alright, thanks for the suggestion